src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java

```diff
@@ -1,24 +1,23 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- * <p>
+ *
  * This software is the confidential and proprietary information of Fundacion Jala
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
  *
  * @author Daniela Santa Cruz
  * @colaborathor Paola Aguilar
- */
-/**
- * @author Daniela Santa Cruz
- * @colaborathor Paola Aguilar
+ * @colaborathor Gustavo Huanca Alconz

```

Comment:
In the code convention we agreed that we do not have to write the collaborators

Reply:
ok